multiqc/modules/multivcfanalyzer/multivcfanalyzer.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from MultiVCFAnalyzer """
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import json
+
+from multiqc.plots import table
+from multiqc.plots import bargraph
+from collections import OrderedDict
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+    """ MultiVCFAnalyzer module """
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='MultiVCFAnalyzer', anchor='multivcfanalyzer',
+        href="https://github.com/alexherbig/MultiVCFAnalyzer",
+        info="""combines multiple VCF files in a coherent way, can produce summary statistics and downstream analysis formats for phylogeny reconstruction.""")
+
+        # Find and load any MultiVCFAnalyzer reports
+        self.mvcf_data = dict()
+
+        # Find and load JSON file
+        for f in self.find_log_files('multivcfanalyzer', filehandles=True):
+            self.parse_data(f)
+
+        # Filter samples
+        self.mvcf_data = self.ignore_samples(self.mvcf_data)
+
+        # Return if no samples found
+        if len(self.mvcf_data) == 0:
+            raise UserWarning
+
+        # Add in extra columns to data file
+        self.compute_perc_hets()
+
+        # Save data output file
+        self.write_data_file(self.mvcf_data, 'multiqc_multivcfanalyzer')
+
+        # Add to General Statistics
+        self.addSummaryMetrics()
+
+        # Add MultiVCFAnalyzer Table Section
+        self.add_section (
+        name = 'MultiVCFAnalyzer analysis results',
+        anchor = 'mvcf_table',
+        helptext = "This is the output from MultiVCFAnalyzer. Please refer to the manual here - https://github.com/alexherbig/MultiVCFAnalyzer for more information on how to interpret data.",
+        plot = self.addTable())
+        
+        # Add MultiVCFAnalyzer Barplot Section
+        self.add_section (
+        name = 'Call statistics barplot',
+        anchor = 'mvcf-barplot',
+        plot = self.addBarplot()
```

Comment:
This is the section that really needs clarification to help understand the output I think.

Reply:
Help text should say:
```
"""
This plot shows the number of SNPs that fall in different categories during filtering, based on: genotyping quality; depth coverage; allele frequency fraction and user-defined position exclusion list.

        * _SNP Calls (all):_ Total number of non-reference calls made
        * _Filtered SNP call:_ Number of non-reference positions excluded by user-supplied list.
        * _Number of Reference Calls:_ Number of reference calls made
        * _Discarded Reference Calls:_ Number of reference positions not reaching genotyping or coverage thresholds
        * _Discarded SNP Call:_ Number of non-reference positions not reaching genotyping or coverage thresholds
        * _No Call:_ Number of positions with no call made as reported by GATK
"""
```

@apeltzer shouldn't 'unhandled Genotypes' also be included (or have you combined this with another column)? The help message here would be: 

```
* _Unhandled Genotypes:_ Number of positions where more than two possible alleles occured and were discarded

```

Reply:
In addition:

@apeltzer shouldn't 'unhandled Genotypes' also be included (or have you combined this with another column)? The help message here would be: 

```
* _Unhandled Genotypes:_ Number of positions where more than two possible alleles occured and were discarded

```